localization/mobile-center-distribute/ru-RU/mobile_center_distribute.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string name="mobile_center_distribute_update_dialog_title">Доступно обновление приложения</string>
```

Comment:
Nitpick: it takes two lines, [shorter version](https://github.com/Microsoft/mobile-center-sdk-android/blob/develop/sdk/mobile-center-distribute/src/main/res/values-ru/mobile_center_distribute.xml#L3) looks better for title